native/source/mixpanel/detail/mixpanel.cpp

```diff
@@ -182,6 +182,18 @@ namespace mixpanel
         return state["distinct_id"].asString();
     }
 
+    std::string Mixpanel::get_alias() const
+    {
+        if (state["alias"].isString())
```

Comment:
wouldn't this throw a null pointer exception if you haven't aliased the user? maybe you can use `string.isNullOrEmpty()`